catalog/video/video-admin-plugin/pom.xml

```diff
@@ -26,13 +26,6 @@
     <properties>
         <timestamp>${maven.build.timestamp}</timestamp>
         <maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
-    
```

Comment:
@glenhein, I added you as a reviewer so that you could take a look at how I've removed powermock from code which you wrote. If you have objections, suggestions, or questions, please let me know.